docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -201,6 +201,41 @@ to the filter:
     ALTER TABLE test_partitioned_table EXECUTE optimize
     WHERE partition_key = 1
 
+expire_snapshots
+~~~~~~~~~~~~~~~~
+
+The ``expire_snapshots`` command removes all snapshots and all related metadata and data files.
+Regularly expiring snapshots is recommended to delete data files that are no longer needed,
+and to keep the size of table metadata small.
+The age of snapshots that will be expired is defined by ``retention_threshold`` parameter.
+
+``expire_snapshots`` can be run as follows:
+
+.. code-block:: sql
+
+  ALTER TABLE test_table EXECUTE expire_snapshots(retention_threshold => '7d')
+
+The ``iceberg.expire_snapshots.min-retention`` config property provides a safety
```

Comment:
drop "config" or use "catalog configuration property" ..

also probably reword the sentence to be specific .. "safety measure" doesnt really spell out what happens .. I think this is the case...  snapshots within min-retention time period are not affected by the procedure..  its basically an override .. 

example scenario: min-retention set to 30 days

call with retention_threshold 21 days
snapshots from 30 days ago and older are deleted .. 

call with 30 days
nothing happens

call with 60 days
snapshots from 60 days ago and older are deleted ..

Reply:
nope, you just can't run the procedure with retention_threshold smaller than iceberg.expire_snapshots.min-retention

Reply:
ok .. well.. then we need to clarify that in the docs. 

And also .. is there an useful error message when you try?

Reply:
yes there is an useful message but it is quite long: `Retention specified (33.00s) is shorter than the minimum retention configured in the system (7.00d). Minimum retention can be changed with iceberg.expire_snapshots.min-retention configuration property or iceberg.expire_snapshots_min_retention session property`